controllers/remote/cluster_cache.go

```diff
@@ -474,7 +474,7 @@ func healthCheckPath(sourceCfg *rest.Config, requestTimeout time.Duration, path
 		return err
 	}
 
-	_, err = restClient.Get().AbsPath(path).Timeout(requestTimeout).Do().Get()
+	_, err = restClient.Get().AbsPath(path).Timeout(requestTimeout).Do(context.TODO()).Get()
```

Comment:
do you know why it adds `context.TODO()` instead of `context.Background()`?

Reply:
I think I forgot to change this part, going to change it now.